<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chess</class>
 <widget class="QWidget" name="Chess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>163</width>
     <height>139</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="createConnectionBtn">
      <property name="text">
       <string>create the connection</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connectServerBtn">
      <property name="text">
       <string>connect to sever</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startBtn">
      <property name="text">
       <string>start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="surrenderBtn">
      <property name="text">
       <string>surrender</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="timeRecord">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>timeRecord</string>
   </property>
  </widget>
  <widget class="QLabel" name="turnShower">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>turnShower</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>your pieces colour</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>round:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>turn to waste:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
